Makefile

```diff
@@ -152,6 +152,9 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+skopeo:
```

Comment:
What's the need for skopeo here?

Reply:
skopeo is required by hack/env.sh

Reply:
Just in case, I think we don't assume local installation will work in a disconnected environment, right?